README.md

```diff
@@ -1,2 +1,52 @@
-# parking-lot-map
-Parking Lot Map
+# Parking Lot Map
+
+The code behind https://parkingreform.org/parking-lot-map/.
+
+The code is fairly simple and we intentionally are not using fancy frameworks like React or Svelte. The main files are `index.html`, `js/script.js`, and `data/*.geojson`. `js/script.js` will load the `.geojson` files to dynamically update `index.html` with all our data.
+
+# How tos
+
+All the commands take place in a _terminal_, a text-based interface for interacting with your computer. On macOS, you can open the "Terminal" app. On Windows, you can use "Git Bash", which will be installed when you install Git below. 
+
+You will also need to install:
+
+* Git, which we use for "version control": https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+* NPM, which is how we run JavaScript and our tests. Use the LTS at https://nodejs.dev/en/download/
```

Comment:
I think it's going to be too hard to explain every single step of how to get started. So I'm thinking that I will pair program with e.g. Thomas to get set up.